src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataUpdater.java

```diff
@@ -48,29 +48,13 @@ protected void updateSingleEntry(BloggerEntry bloggerEntry, UpdateSummary update
             Blog bloggerWithSameJsonId = existingBloggerByJsonId.get();
             updateBloggerIfThereAreSomeChanges(bloggerEntry, updateSummary, bloggerWithSameJsonId);
         } else {
-            blogRepository.save(Blog.builder()
-                    .jsonId(bloggerEntry.getJsonId())
-                    .author(bloggerEntry.getName())
-                    .rss(StringUtils.lowerCase(bloggerEntry.getRss()))
-                    .url(syndFeedFactory.urlFromRss(
-                        bloggerEntry.getRss()).orElse(null)
-                    )
-                    .twitter(bloggerEntry.getTwitter())
-                    .dateAdded(nowProvider.now())
-                    .blogType(bloggerEntry.getBlogType())
-                    .active(true)
-                    .build());
-            updateSummary.recordCreated();
+            createNewBlogger(bloggerEntry, updateSummary);
         }
     }
 
     private void updateBloggerIfThereAreSomeChanges(BloggerEntry bloggerEntry,
                                                     UpdateSummary updateSummary,
                                                     Blog existingBlogger) {
-        bloggerEntry.setUrl(
```

Comment:
I think [`BloggersDataUpdaterSpec`](https://github.com/tdziurko/jvm-bloggers/blob/master/src/test/groovy/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataUpdaterSpec.groovy) unit test is too weak if it was green even after removing the line. We should consider to make the test more robust to cover that case.

I'm also thinking about adding some mutation testing capabilities (not in this but in a separate PR) using [_PIT_](http://pitest.org/). It will definitely detect such case before it occurs in real. There is also _Gradle_ plugin for it. I'm using it in every project from some time and it's a real game changer regarding quality of unit tests. What do you think? If you agree I can create an issue for that.